custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -269,8 +268,13 @@ def fit(self, X, y, output_dir, class_order=None, row_weights=None):
                 X, y, output_dir, class_order=class_order, row_weights=row_weights
             )
         else:
+            hooks = [
+                "{}: {}".format(hook, fn is not None) for hook, fn in self._custom_hooks.items()
+            ]
             raise DrumCommonException(
-                "fit() method must be implemented in the 'custom.py' in the provided code_dir: '{}'".format(
-                    self._model_dir
+                "\nfit() method must be implemented in a file named 'custom.py' in the provided code_dir: '{}' \n"
+                "Here is a list of files in this dir. {}\n"
+                "Here are the hooks your model has: {}".format(
```

Comment:
Maybe better: `Here are the hooks your code implements`
as I understand, we can not use word `model` for training, (for the same reason we switched form `model_dir` to `code_dir`)

Reply:
I thought about that and chose this way on purpose, I think its good for people to see the full range of hooks. I can fix the word model in a followup if thats all I need to change.

Reply:
List of hooks is ok. I'm talking only about `model` word. And anyway you'll need to rebase on master I think?